purchase_rfq_number/models/res_company.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 ProThai Technology Co.,Ltd. (http://prothaitechnology.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    keep_name_po = fields.Boolean(
+        string="Use Same Enumeration",
+        help="If this is unchecked, purchase rfq use a different sequence from "
+        "Purchase orders",
+        default=True,
+    )
+    auto_attachment_rfq = fields.Boolean(
+        string="Auto Attachment RFQ",
```

Comment:
Print RFQ as attachment to confirmed order.
```suggestion
        string="Print RFQ as attachment to confirmed order",
```